src/main/webapp/index.html

```diff
@@ -334,10 +436,20 @@ <h2 class="font-weight-900 text-dark text-roboto text-center">
 <script src="assets/js/global-custom-js.js"></script>
 <!-- Script for static navbar and footer -->
 <script src="assets/js/navigation.js"></script>
+<script id="star-note-content" type="text/html">
+    <p class="small text-muted text-left mb-0"> *Our mentors and speakers engage in mentoring and speaking voluntarily
+        in their leisure time<br> and we don't have a direct affiliation with the mentioned organisations</p>
+</script>
 <script>
     $(function () {
         loadNavAndFooter('assets/content/static')  //relative path to content directory
         // link to the navbar, footer content goes here
+    });
+
+    $(document).ajaxComplete(function () {
+        $("#star-note").html(
+            $("#star-note-content").html()
+        )
     })
```

Comment:
Why do we load this using a jquery function instead of just putting it on the page?

Reply:
It has to be inside the footer. But can't directly add it to the footer because we don't need it to display on other pages

Reply:
@Gravewalker666 is this sorted?

Reply:
@Piumal1999 Is this the best way to do that.
Do we have a way to add a custom footer?

Reply:
I think this is the best way. Because using another footer instead of the using the current dynamic one may not be much effective.

Reply:
Alright cool!!